python/example_code/lookoutvision/inference.py

```diff
@@ -95,50 +83,204 @@ def download_from_s3(s3_resource, photo):
+        :param prediction: The DetectAnomalyResult object returned from DetectAnomalies
+        :param confidence_limit: The minimum acceptable confidence. Float value between 0 and 1.
+        :return: True if the error condition indicates an anomaly, otherwise False.
+        """
+
+        reject = False
+
+        logger.info("Checking classification for %s", image)
+
+        if prediction['IsAnomalous'] and prediction['Confidence'] >= confidence_limit:
+            reject = True
+            reject_info=(f"Rejected: Anomaly confidence ({prediction['Confidence']:.2%}) is greater"
+                    f" than limit ({confidence_limit:.2%})")
+            logger.info("%s", reject_info)
+
+        if not reject:
+            logger.info("No anomalies found.")
+        return reject
+
+    @staticmethod
+    def reject_on_anomaly_types(image, prediction, confidence_limit, anomaly_types_limit):
+        """
+        Checks if the number of anomaly types is greater than than the anomaly types
+        limit and if the prediction confidence is greater than the confidence limit.
+        :param image: The name of the image file that was analyzed.
+        :param prediction: The DetectAnomalyResult object returned from DetectAnomalies
+        :param confidence: The minimum acceptable confidence. Float value between 0 and 1.
+        :param anomaly_types_limit: The maximum number of allowable anomaly types (Integer).
+        :return: True if the error condition indicates an anomaly, otherwise False.
+        """
+
+        logger.info("Checking number of anomaly types for %s",image)
+
+        reject = False
+
+        if prediction['IsAnomalous'] and prediction['Confidence'] >= confidence_limit:
+
+            anomaly_types = {anomaly['Name'] for anomaly in prediction['Anomalies']\
+                if anomaly['Name'] != 'background'}
+
+            if len (anomaly_types) > anomaly_types_limit:
+                    reject = True
+                    reject_info = (f"Rejected: Anomaly confidence ({prediction['Confidence']:.2%}) "
+                    f"is greater than limit ({confidence_limit:.2%}) and "
+                    f"the number of anomaly types ({len(anomaly_types)-1}) is "
+                    f"greater than the limit ({anomaly_types_limit})")
+
+                    logger.info("%s", reject_info)
+
+        if not reject:
+            logger.info("No anomalies found.")
+        return reject
+
+    @staticmethod
+    def reject_on_coverage(image, prediction, confidence_limit, anomaly_label, coverage_limit):
+        """
+        Checks if the coverage area of an anomaly is greater than the coverage limit and if
+        the prediction confidence is greater than the confidence limit.
+        :param image: The name of the image file that was analyzed.
+        :param prediction: The DetectAnomalyResult object returned from DetectAnomalies
+        :param confidence_limit: The minimum acceptable confidence (float 0-1).
```

Comment:
Capitalize "Float"

Reply:
Left as is to match python 'float' type.